### PR TITLE
Resolve #5404 - Add a Languages submenu item to open UDL folder.

### LINKED ISSUE #5404: [Request]: Open UDL Language(s) Folder...
### Description of the Issue

Hello,
I would like to see this feature in the `Language` menu, in the `Define your language...` section. The same as `Plugins` > `Open Plugins Folder...` but for opening the `userDefineLangs` folder.

### Debug Information

Notepad++ v7.6.4   (64-bit)
Build time : Mar  6 2019 - 02:58:24
Path : D:\Install\Office Programs\Notepad++\notepad++.exe
Admin mode : ON
Local Conf mode : ON
OS : Windows 7 (64-bit)
Plugins : AutoSave.dll ComparePlugin.dll VisualStudioLineCopy.dll 

### Other Info

I also open the same request at the Community: [[Request]: Open Language(s) Folder...](https://notepad-plus-plus.org/community/topic/17271/request-open-language-s-folder)

### PR DESCRIPTION
**Pull request by KrzysztofSzewczyk**
This pull request will resolve issue #5404

### CODE REVIEW

**Comment by rddim** [comment]
Hello and thank you very much!

There is an issue when the `Settings > Preferences... > Language > Make language menu compact` is **checked** the `Open UDL folder...` is not available in the menu. If the `Make language menu compact` is **unchecked**, then the option is available.

enabled compact menu
npp_compact_lang

disabled compact menu
npp_no_compact_lang

### Debug Info
Notepad++ v7.8.1   (64-bit)
Build time : Nov 10 2019 - 11:58:45
Path : D:\Install\Office Programs\npp.7.8.1.bin.x64\Notepad++.x64.Unicode Release.exe
Admin mode : ON
Local Conf mode : ON
OS Name : Windows 7 Professional (64-bit) 
OS Build : 7601.0
Plugins : none

### Other Info
Used appveyor build [7.8.3468](https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/28747708/job/7cgjbjdpcpucvejk/artifacts)

**Comment by KrzysztofSzewczyk** [comment]
If it shall be fixed (and it isn't intened behaviour accordingly), I'll take on it shortly. Good catch!

**Comment by KrzysztofSzewczyk** [comment]
No idea on how to tackle on this one, any ideas? I've been lurking NP++ quite a bit, yet I can't really tell how can I override this behaviour without ruining the code.

**Comment by chcg** [comment]
@KrzysztofSzewczyk 
Isn't it sufficient to add the new entry also to compact list at https://github.com/notepad-plus-plus/notepad-plus-plus/blob/c741f83bde6028273e72e8775e4e0854a70dbea6/PowerEditor/src/Notepad_plus.rc#L920 ?

**Comment by KrzysztofSzewczyk** [comment]
Well, done :p

**Comment by rddim** [comment]
Perfect, now the option is available in both ways :)

**Comment by KrzysztofSzewczyk** [comment]
@donho 
Can I get a review?

File: PowerEditor/src/NppCommands.cpp [hunk 1]
```diff
@@ -3048,6 +3048,13 @@ void Notepad_plus::command(int id)
 			}
 		}
         break;
+		
+		case IDM_LANG_OPENUDLDIR: {
```
Comment (donho) [inline PowerEditor/src/NppCommands.cpp]: https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#good

Comment (donho) [reply PowerEditor/src/NppCommands.cpp]: And the wrong indentation.

**Comment by donho** [comment]
But I'm not sure it's the right place for this feature.

**Comment by KrzysztofSzewczyk** [comment]
So it's not getting merged?

**Comment by LordNyriox** [comment]
@KrzysztofSzewczyk: 

> So it's not getting merged? 

This PR is already merged.  See commit [<https://github.com/notepad-plus-plus/notepad-plus-plus/commit/2458812a42e91dbcc531401926290ecb19984bac>]. 

@donho always manually merges PRs, using some form of external tool that Github does not automatically detect.

**Pull request closed without merging**